src/perturbopy/postproc/calc_modes/spectral_cumulant.py

```diff
@@ -0,0 +1,88 @@
+import numpy as np
+import h5py as h5
+import matplotlib.pyplot as plt
+from perturbopy.postproc.calc_modes.calc_mode import CalcMode
+
+
+class SpectralCumulant(CalcMode):
+    """
+    A class to handle and analyze spectral functions computed using the cumulant expansion approach.
+
+    Attributes:
+    - T (numpy.ndarray): Array of temperatures corresponding to the spectral data.
+    - w (numpy.ndarray): Array of energy values (ω) in electron volts (eV).
+    - dw (float): Energy step size for the ω grid in eV.
+    - Akw (numpy.ndarray): Spectral function data, indexed by k-point, band, temperature, and ω.
+    """
+    def __init__(self, prefix):
```

Comment:
We have to use a standard already implemented in PerturboPy:

1. Define a class method:
```
    @classmethod
    def from_hdf5_yaml(cls, cdyna_path, tet_path, yaml_path='pert_output.yml'):
```
2. Initialize your class with __init__:
```
    @classmethod
    def from_hdf5_yaml(cls, cdyna_path, tet_path, yaml_path='pert_output.yml'):
```
See dyna_run.py for example